src/app/create-guild/components/CreateGuildFormProvider.tsx

```diff
@@ -0,0 +1,25 @@
+"use client";
+
+import { type CreateGuildForm, GuildSchema } from "@/lib/schemas/guild";
+import { zodResolver } from "@hookform/resolvers/zod";
+import type { PropsWithChildren } from "react";
+import { FormProvider, useForm } from "react-hook-form";
+
+const defaultValues = {
+  name: "",
+  imageUrl: "",
+  urlName: "test",
+  contact: "",
+} satisfies CreateGuildForm;
+
+const CreateGuildFormProvider = ({ children }: PropsWithChildren) => {
+  const methods = useForm<CreateGuildForm>({
+    mode: "all",
```

Comment:
`all` might be obtrusive, how about we try `onTouched`?

Reply:
Oooh, that's a great idea! 💯